README.md

```diff
@@ -11,13 +11,15 @@ capa detects capabilities in executable files.
 You run it against a PE, ELF, .NET module, shellcode file, or a sandbox report and it tells you what it thinks the program can do.
 For example, it might suggest that the file is a backdoor, is capable of installing services, or relies on HTTP to communicate.
 
-Check out our capa blog posts:
-- [Dynamic capa: Exploring Executable Run-Time Behavior with the CAPE Sandbox](https://www.mandiant.com/resources/blog/dynamic-capa-executable-behavior-cape-sandbox)
-- [capa v4: casting a wider .NET](https://www.mandiant.com/resources/blog/capa-v4-casting-wider-net) (.NET support)
-- [ELFant in the Room – capa v3](https://www.mandiant.com/resources/elfant-in-the-room-capa-v3) (ELF support)
-- [capa 2.0: Better, Stronger, Faster](https://www.mandiant.com/resources/capa-2-better-stronger-faster)
-- [capa: Automatically Identify Malware Capabilities](https://www.mandiant.com/resources/capa-automatically-identify-malware-capabilities)
+This repo hosts the:
+- core capa source code
+- [capa Web Explorer](https://mandiant.github.io/capa/#/) to interactively inspect capa results in your browser
```

Comment:
general comment: we need to figure out and be consistent with what we're calling this thing.

some options:
  - capa web explorer
  - capa explorer web
  - capa explorer web application
  - web capa explorer
  - capachef

once we've decided, we should go back through all the source code and make things consistent.